curriculum/challenges/english/25-front-end-development/workshop-piano/612ea97df5742154772f312e.md

```diff
@@ -0,0 +1,110 @@
+---
+id: 612ea97df5742154772f312e
+title: Step 17
+challengeType: 0
+dashedName: step-17
+---
+
+# --description--
+
+Give the `.key.black--key::after` a `position` property set to `absolute` and a `left` property set to `-18px`.
```

Comment:
```suggestion
For the `.key.black--key::after` selector, add a `position` property set to `absolute` and a `left` property set to `-18px`. This will position the pseudo-element absolutely relative to its nearest positioned ancestor, allowing for precise placement on the page.

```